up42/storage.py

```diff
@@ -236,15 +255,20 @@ def get_orders(
         if sortby not in allowed_sorting_criteria:
             raise ValueError(f"sortby parameter must be one of {allowed_sorting_criteria}!")
         sort = f"{sortby},{'desc' if descending else 'asc'}"
-        url = f"{self.auth._endpoint()}/workspaces/{self.workspace_id}/orders?format=paginated&sort={sort}"
-        if order_type is not None:
-            if order_type in ["TASKING", "ARCHIVE"]:
-                url += f"&type={order_type}"
-            else:
-                logger.warning("order_type should be TASKING or ARCHIVE. Ignoring this filter.")
-        if tags is not None:
-            for tag in tags:
-                url += f"&tags={tag}"
+        base_url = f"{self.auth._endpoint()}/v2/orders"
+
+        params = {
```

Comment:
Test coverage is missing for all new branches added

Reply:
This is unaddressed, but please ignore it since we need to revise all tests anyway

Reply:
I have added some new tests, according to our conversation from today.